corehq/apps/registration/static/registration/js/register_new_user.js

```diff
@@ -2,6 +2,29 @@
 $(function () {
     var initial_page_data = hqImport('hqwebapp/js/initial_page_data').get;
 
+    $('#js-start-trial').click(function (e) {
+        e.preventDefault();
+        $('#registration-start-container').addClass('hide');
+        $('#registration-form-container').fadeIn();
+    });
+
+    var kissmetrics = hqImport('analytix/js/kissmetrix');
+    kissmetrics.whenReadyAlways(function () {
```

Comment:
Why does this need `whenReadyAlways`? `track` is already promise-based: https://github.com/dimagi/commcare-hq/blob/33ac8470f632053f5b69a3a38b3af4521e5090fb/corehq/apps/analytics/static/analytix/js/kissmetrix.js#L124